internal/pkg/cli/cli_integration_test.go

```diff
@@ -0,0 +1,63 @@
+// +build integration
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	manifestPath           = "autoscaling-svc-manifest.yml"
+	wantedCFNTemplatePath  = "autoscaling-svc-cf.yml"
+	wantedCFNParameterPath = "autoscaling-svc-cf.params.json"
+
+	appName  = "my-app"
+	envName  = "test"
+	imageURL = "mockImageURL"
+	imageTag = "latest"
+)
+
+func Test_Autoscaling_Integration(t *testing.T) {
+	path := filepath.Join("testdata", manifestPath)
+	wantedManifestBytes, err := ioutil.ReadFile(path)
+	require.NoError(t, err)
+	mft, err := manifest.UnmarshalService(wantedManifestBytes)
+	require.NoError(t, err)
+	v, ok := mft.(*manifest.LoadBalancedWebService)
+	require.Equal(t, ok, true)
+	serializer, err := stack.NewHTTPSLoadBalancedWebService(v, envName, appName, stack.RuntimeConfig{
```

Comment:
This is an integration test really for the `stack` package since the methods that we're invoking is on a `stack.LoadBalancedWebService` object.

Can we move the tests and the testdata files to the `stack` package?